embeddings/pipeline/hugging_face_sequence_labeling.py

```diff
@@ -33,14 +36,21 @@ def __init__(
         hidden_size: int,
         evaluation_mode: str = "conll",
         tagging_scheme: Optional[str] = None,
+        sample_missing_splits: Tuple[float, float] = (0.1, 0.1),
```

Comment:
Same as previously: In my view sampling in pipelines should be optional instead of forcing sampling.